### DISCUSSION TITLE
Can HTML subtables without captions be rendered as PDFs?

### DISCUSSION THREAD

**Discussion by IULibScholComm** [quarto-dev/quarto-cli#8655] [Q&A] [answered]
### Description

Can subtables in HTML (rather than markdown) be rendered as PDFs when those subtables lack subcaptions? 

When I do so, I get a lua error: "C:/Program Files/Quarto/share/filters/main.lua:17131: attempt to index a nil value" (see below)

Error running filter C:/Program Files/Quarto/share/filters/main.lua:
C:/Program Files/Quarto/share/filters/main.lua:17131: attempt to index a nil value
stack traceback:
        C:/Program Files/Quarto/share/filters/main.lua:16834: in function 'latexPanel'
        C:/Program Files/Quarto/share/filters/main.lua:16788: in field 'render'
        C:/Program Files/Quarto/share/filters/main.lua:835: in local 'filter_fn'
        C:/Program Files/Quarto/share/filters/main.lua:302: in function <C:/Program Files/Quarto/share/filters/main.lua:292>
        (...tail calls...)
        [C]: in ?
        [C]: in method 'walk'
        C:/Program Files/Quarto/share/filters/main.lua:224: in function <C:/Program Files/Quarto/share/filters/main.lua:214>
        (...tail calls...)
        C:/Program Files/Quarto/share/filters/main.lua:936: in local 'callback'
        C:/Program Files/Quarto/share/filters/main.lua:954: in upvalue 'run_emulated_filter_chain'
        C:/Program Files/Quarto/share/filters/main.lua:990: in function <C:/Program Files/Quarto/share/filters/main.lua:987>
stack traceback:
        C:/Program Files/Quarto/share/filters/main.lua:224: in function <C:/Program Files/Quarto/share/filters/main.lua:214>
        (...tail calls...)
        C:/Program Files/Quarto/share/filters/main.lua:936: in local 'callback'
        C:/Program Files/Quarto/share/filters/main.lua:954: in upvalue 'run_emulated_filter_chain'
        C:/Program Files/Quarto/share/filters/main.lua:990: in function <C:/Program Files/Quarto/share/filters/main.lua:987>

Here is my minimal example: 
````qmd
---
title: "test"
format: pdf
---

:::: {#tbl-test layout-ncol="2"}

::: {#tbl-test1}

```{=html}
<table>
  <thead>
    <tr>
      <th>test</th>
      <th>test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>test</td>
      <td>test</td>
    </tr>
  </tbody>
</table>
```

:::

::: {#tbl-test2}

```{=html}
<table>
  <thead>
    <tr>
      <th>test</th>
      <th>test</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>test</td>
      <td>test</td>
    </tr>
  </tbody>
</table>
```

:::

Test
::::
````

When the subtables have subcaptions, everything works fine, but when they lack subcaptions, I get the lua error. 

I'm using Quarto 1.4.59 on Windows 10 Enterprise. 

Thank you for your help!

**Comment by cscheid** [comment, marked as answer]
This is a bug. Thank you for the clean repro!

**Reply by cderv** [reply]
Tracked now in 
* https://github.com/quarto-dev/quarto-cli/issues/8656
